code/game/turfs/simulated/floor/plating.dm

```diff
@@ -159,18 +159,38 @@
 	nitrogen = 0
 	temperature = TCMB
 
+/turf/simulated/floor/plating/abductor
+	name = "alien floor"
+	icon_state = "alienpod1"
+
+/turf/simulated/floor/plating/abductor/New()
+	..()
+	icon_state = "alienpod[rand(1,9)]"
+
 /turf/simulated/floor/plating/lava
+	name = "lava"
 	icon_state = "lava"
+	baseturf = /turf/simulated/floor/plating/lava //lava all the way down
+	slowdown = 2
 
 /turf/simulated/floor/plating/lava/airless
 	oxygen = 0
 	nitrogen = 0
 	temperature = TCMB
 
-/turf/simulated/floor/plating/abductor
-	name = "alien floor"
-	icon_state = "alienpod1"
-
-/turf/simulated/floor/plating/abductor/New()
-	..()
-	icon_state = "alienpod[rand(1,9)]"
\ No newline at end of file
+/turf/simulated/floor/plating/lava/Entered(atom/movable/AM)

```

Comment:
I can enter the tile, resist, and survive like this.
When sticky made his lava tile (Which I dont know if that was ever merged...) we encouraged him to add a ```process()``` that simply called code similar to what you have below, but on the contents list of the turf.
(so long as there were contents, obviously) with some extra code to add/remove it from the processing list (```Entered()```) for further robust code.